content/pages/people.md

```diff
@@ -1,8 +1,29 @@
-# Project Pythia core team
-
-- John Clyne, NCAR
-- Ryan May, Unidata
-- Kevin Paul, NCAR
-- Matthew C. Long, NCAR
-- Brian E. J. Rose, U. Albany
-- Kevin Tyle, U. Albany
+# Project Pythia Team
+
+## Project Administrators
+
+Project Pythia is adminstrated by the Principal Investigators of the
+NSF EarthCube awards \#2026899 (UCAR) and \#2026863 (University at Albany).
+
+- John Clyne (NCAR) [[@clyne](https://github.com/clyne)]
+- Matthew C. Long (NCAR) [[@matt-long](https://github.com/matt-long)]
+- Ryan May (Unidata) [[@dopplershift](https://github.com/dopplershift)]
+- Kevin Paul (NCAR) [[@kmpaul](https://github.com/kmpaul)]
+- Brian E. J. Rose (University at Albany) [[@brian-rose](https://github.com/brian-rose])]
```

Comment:
There's an extra bracket here which is breaking the link.

Reply:
Good catch!  I missed that.